lexis/data/frame.fbs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Jafet.VillafrancaDiaz@epfl.ch
+//
+namespace lexis.data;
+
+// Event containing the data frame range information. It is based on the same
+// units as the zerobuf.render.Frame, i.e. number of frames.
+// It may be used as a base event that defines the available range of frames
+// coming from a certain data source, being also possible to use the
+// zerobuf.render.Frame event at the same time, which would define the frame
+// subrange used by a rendering animation to loop within.
+
+table Frame
```

Comment:
No, probably not. Only this one clearly represent a frame range with a start/end and it is documented as such. The render/frame.fbs contains in addition the current position and animation delta, which makes it something more than a simple frame range (not sure what - I remember there were discussions to split it into smaller pieces at some point).